cns/NetworkContainerContract.go

```diff
@@ -239,6 +248,10 @@ func (p *podInfo) OrchestratorContext() (json.RawMessage, error) {
 	return jsonContext, nil
 }
 
+func (p *podInfo) IsMultitenant() bool {
+	return p.Multitenant
```

Comment:
do we need a Getter for this? The struct member is already public

Reply:
im following the convention for other podInfo's field getters, but yeah this makes sense.

Reply:
we do, because the object is exclusively accessed through the interface (note the struct is private `podInfo`)

Reply:
Is there a reason why we are accessing `podInfo` object through `PodInfo` interface? Because I notice that the only thing that is implementing this interface is `podInfo` itself.

Reply:
IIRC it was to force usage of the `Key()` method, which does different things based on how the `podInfo` was created, rather than allow direct access to the fields